model/src/main/java/org/cbioportal/genome_nexus/model/VariantAnnotation.java

```diff
@@ -263,6 +263,7 @@ public void setMutationAssessorAnnotation(MutationAssessorAnnotation mutationAss
         this.mutationAssessorAnnotation = mutationAssessorAnnotation;
     }
 
+    @Field(value="my_variant_info")
```

Comment:
Not sure if we need this one, since `my_variant_info` field should not exist in the `vep.annotation` collection.